### DISCUSSION TITLE
Dark theme is hard to see

### DISCUSSION THREAD

**Discussion by tsuda-m** [gitextensions/gitextensions#11920] [Q&A] [answered]
The dark theme is hard to see.
The light theme is still present in places and is halfway through the theme.
![image](https://github.com/user-attachments/assets/393e8c70-0767-4244-9793-9e0b77cb260d)

I tried to get the dark theme css file that was there before (probably before V4.0) and apply it, but it was still the same and hard to see.

Is there any way to use the same dark theme in the current version as before?
(For example, there is a user custom css file created by someone else, etc.)

**Comment by RussKie** [comment, marked as answer]
Duplicate of https://github.com/gitextensions/gitextensions/issues/9191

Windows high contrast dark Mode is required for the HE theme. (That is ugly.)

It is not technically possible to override colors in .NET 8 as it was in .NET Framework. There is some experimental support in the upcoming .NET 9, but there seem to be issues, so it is not known if it is possible.
